### DISCUSSION TITLE
Communications

### DISCUSSION THREAD

**Discussion by ClareBellingham** [nvs-vocabs/R28#3] [General]
I have emailed Ocean-ops at metadata@ocean-ops.org to find out where they are up to with their clean up of the controller board type and whether they are recording only board type or also including a description

**Comment by anthoninlize** [comment]
Hi Clare, 

Replying to your email here then. @mbelbeoch  started documenting and cleaning the controller board type in the Google drive document [here](https://docs.google.com/spreadsheets/d/1Aw8B7FFUjG4e9MveD5qqvI7ZbB-z_x32IQnpHZbXVZA/edit#gid=1206152030). This would require further work but that would be a good starting base. 

So we store indeed the name of the controller board type ('JCOMMOPS audit' column) but also its serial reference.
